flink-connector-postgres-cdc/src/main/java/com/ververica/cdc/connectors/postgres/source/PostgresSourceBuilder.java

```diff
@@ -215,6 +215,23 @@ public PostgresSourceBuilder<T> debeziumProperties(Properties properties) {
         return this;
     }
 
+    /**
+     * scan.incremental.close-idle-reader.enabled
+     *
+     * <p>Whether to close idle readers at the end of the snapshot phase. This feature depends on
+     * FLIP-147: Support Checkpoints After Tasks Finished. The flink version is required to be
+     * greater than or equal to 1.14, and the configuration <code>
+     * 'execution.checkpointing.checkpoints-after-tasks-finish.enabled'</code> needs to be set to
+     * true.
+     *
+     * <p>See more
+     * https://cwiki.apache.org/confluence/display/FLINK/FLIP-147%3A+Support+Checkpoints+After+Tasks+Finished.
+     */
+    public PostgresSourceBuilder<T> closeIdleReaders(boolean closeIdleReaders) {
```

Comment:
This should be a API change feature?

Reply:
There is no API change, here is something omitted here at 2.3

Reply:
cc @GOODBOY008